packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -202,7 +202,7 @@ function normalizeEntry(
 			let contents = entry.value.contents;
 			// If this blob has to be normalized or it's a GC blob, parse and sort the blob contents first.
 			if (
-				config?.blobsToNormalize?.includes(entry.path) ||
+				config?.blobsToNormalize?.includes(entry.path) ??
```

Comment:
Same bug with `boolean | undefined` and `or` vs coalesce.

Reply:
fixed. It's a _little_ frightening to me that tests continue to pass with these logic changes.

Reply:
Agree. Also a little frightening to me that the linter applies them as fixes when they're behavior changes. Probably a bug there?